src/System.Security.Permissions/src/System/Security/CodeAccessPermission.cs

```diff
@@ -0,0 +1,20 @@
+namespace System.Security
+{
+    public abstract partial class CodeAccessPermission : System.Security.IPermission, System.Security.ISecurityEncodable, System.Security.IStackWalk
+    {
+        protected CodeAccessPermission() { }
+        public void Assert() { }
+        public abstract System.Security.IPermission Copy();
+        public void Demand() { }
+        public void Deny() { }
```

Comment:
I'm wondering what the right default is here.  The idea behind these stubs is that CAS isn't actually enforced, so Demands (which effectively say "throw an exception if I don't have this ability") should be a nop, because all such permissions are implicitly granted.  But Deny is different.  Deny effectively says "even if I have this ability, pretend that I don't so that all subsequent demands for it will fail"... which suggests that it could be used to actually impact how our nop Demand behaves.  That makes me think we may want to have the implementation of Deny throw a PlatformNotSupprotedException rather than being a nop. (Assert is ok as a nop I believe, as it effectively tells the system to pretend that a permission that wasn't otherwise granted was actually granted.)
cc: @bartonjs, @morganbr, @blowdart, @danmosemsft